File: src/BukkitPE/event/player/PlayerBucketFillEvent.php
<?php
namespace BukkitPE\event\player;

use BukkitPE\block\Block;
use BukkitPE\item\Item;
use BukkitPE\Player;

class PlayerBucketFillEvent extends PlayerBucketEvent{
	public static $handlerList = null;

	public function __construct(Player $who, Block $blockClicked, $blockFace, Item $bucket, Item $itemInHand){
		parent::__construct($who, $blockClicked, $blockFace, $bucket, $itemInHand);
	}
}

File: src/main/java/net/BukkitPE/event/player/PlayerBucketFillEvent.java
package net.BukkitPE.event.player;

import net.BukkitPE.Player;
import net.BukkitPE.block.Block;
import net.BukkitPE.event.HandlerList;
import net.BukkitPE.item.Item;

public class PlayerBucketFillEvent extends PlayerBucketEvent {
    private static final HandlerList handlers = new HandlerList();

    public PlayerBucketFillEvent(Player who, Block blockClicked, int blockFace, Item bucket, Item itemInHand) {
        super(who, blockClicked, blockFace, bucket, itemInHand);
    }

    public static HandlerList getHandlers() {
        return handlers;
    }

}
